HyperVExtension/src/HyperVExtension/Helpers/BytesHelper.cs

```diff
@@ -1,12 +1,18 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
 using System.Globalization;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.Shell;
 
 namespace HyperVExtension.Helpers;
 
 public static class BytesHelper
 {
+    private const int ByteToStringBufferSize = 16;

```

Comment:
Seems like you're only using `ByteToStringBufferSize` in one location.  Maybe move this const into `ConvertBytesToString`

Reply:
I did it this way since it doesn't change so figured I'd make it const instead of needing to make it stack variable everytime I call the method